### PR TITLE
Setup webhook dispatcher

### PR DESCRIPTION
**Pull request by RaphaelVogel**
relates to https://github.wdf.sap.corp/kubernetes/cc-config/pull/49

### CODE REVIEW

File: model/concourse.py [hunk 1]
```diff
@@ -70,6 +70,9 @@ def image_pull_secret(self):
     def tls_secret_name(self):
         return self.raw.get('tls_secret_name')
 
+    def webhook_dispatcher(self):
```
Comment (ccwienk) [inline model/concourse.py]: add to required attributes, please

Comment (ccwienk) [reply model/concourse.py]: also: why does concourse "own" the dispatcher? should it not be the other way around? the dispatcher will interact (based upon concouse job mappings) with concourse instance(s) and github repositories

File: concourse/setup.py [hunk 2]
```diff
@@ -560,6 +564,38 @@ def deploy_delaying_proxy(
...
+
+    service_helper = ctx.service_helper()
+    deployment_helper = ctx.deployment_helper()
+    namespace_helper = ctx.namespace_helper()
+    ingress_helper = ctx.ingress_helper()
+
+    namespace_helper.create_if_absent(namespace)
+
+    webhook_service = generate_webhook_dispatcher_service()
+    webhook_deployment = generate_webhook_dispatcher_deployment(
```
Comment (ccwienk) [inline concourse/setup.py]: general remark: would it make sense to separate the creation of deployment manifests and their appliance (to better support local development scenarios)? for a first version, it is certainly ok to not do this.

**Comment by RaphaelVogel** [comment]
We go for Helm charts

**Pull request closed without merging**